src/customer-ontology.ttl

```diff
@@ -0,0 +1,91 @@
+@prefix : <http://bibsysdev.github.io/src/customer-ontology#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Customer-related concepts
+
+:Customer a rdf:Class ;
+  rdfs:comment "Customer institution in NVA" .
+
+:VocabularySetting a rdf:Class ;
+  rdfs:comment "Setting for a controlled vocabulary for a customer institution" .
+
+:VocabularyStatus a rdf:Class ;
+  rdfs:comment "Status for a controlled vocabulary for a customer institution" .
+
+:name a rdf:Property ;
+  rdfs:comment "The name of the customer institution" ;
+  rdfs:domain :Customer ;
+  rdfs:range xsd:string .
```

Comment:
Check if the ranges are xsd:string or rdf:langString for all string props

Reply:
They are all xsd:string right now, but maybe some really should be rdf:langString?